client/partials/sensors.html

```diff
@@ -0,0 +1,58 @@
+<!doctype html>
+<style>
```

Comment:
I'll create a separate style sheet for the sensors in the client/sass directory, and we can add these styles to it.